synthnn/core/musical_synthesis.py

```
@@ -6,14 +6,15 @@
 """
 
 import numpy as np
-from typing import Dict, List, Tuple, Optional, Union, Callable
+from typing import Dict, List, Tuple, Optional, Union, Callable, Any
```

Ruff (0.11.9)

`typing.Dict` is deprecated, use `dict` instead

(UP035)